conversions/Forwards/src/main/java/com/agentdid127/resourcepack/forwards/impl/PackMetaConverter.java

```diff
@@ -32,40 +34,50 @@ public void convert(Pack pack) throws IOException {
         Path file = pack.getWorkingPath().resolve("pack.mcmeta");
         if (!file.toFile().exists())
             return;
-        if (version >= Util.getVersionProtocol(packConverter.getGson(), "1.20"))
+        
+        int versionInt = 4;
+	
+        // Possible TODO: Make this JSON? Possibly use protocol.json, but update it.
```

Comment:
Protocol JSON is for Minecraft PRotocol versions, not Resource Pack MCMeta versions. See https://minecraft.wiki/w/Protocol_version for more info

Reply:
Yea I know but I meant like a combined data file with protocol versions, pack versions, etc